test/seqan3-test.cmake

```diff
@@ -126,7 +126,7 @@ macro (seqan3_require_benchmark)
         gbenchmark_project
         PREFIX gbenchmark_project
         GIT_REPOSITORY "https://github.com/google/benchmark.git"
-        GIT_TAG "v1.4.1"
+        GIT_TAG "b8ca0c42179b7b5d656494e61dda8b861057122f"
```

Comment:
Why are switching away from a release?

Reply:
`benchmark::Counter::kIsIterationInvariantRate` is only available in a more recent version (from August 2018, last release was May 2018).

I asked nicely for a new version number. :angel:

Reply:
and?